src/components/Kicker/Kicker.scss

```diff
@@ -0,0 +1,8 @@
+@import '~@quartz/styles/scss/color-scheme';
+@import '~@quartz/styles/scss/fonts';
+
+.container {
+	@include font-maison-extended-700-1;
+
+	text-transform: uppercase;
```

Comment:
This is already in the above mixin; turned out we would never use that mixin without it being all-caps https://github.com/Quartz/styles/pull/26